### PR TITLE
Add ecosystem page for website

### LINKED ISSUE #699: Website Update
```[tasklist]
### Tasks
- [ ] https://github.com/microsoft/autogen/issues/687
- [ ] https://github.com/microsoft/autogen/issues/700
- [ ] https://github.com/microsoft/autogen/issues/799
```

### LINKED ISSUE #799: Add link to AutoGen_Playground created by Li on AutoGen website
Comment from Tonic on discord: " big thanks to @Li_Jiang for the AWESOME autogen demo : https://huggingface.co/spaces/thinkall/AutoGen_Playground (using the new panel template for gui, it's really really really well done!)"

### PR DESCRIPTION
**Pull request by skzhang1**
## NEED HELP !!

1. Does anyone know how to render the posts in a style like https://boxyhq.com/blog with icons? 
2. Enrich the ecosystem page (application + examples)
3. Any other suggestions regarding the website.

## Why are these changes needed?

1. Add ecosystem page 
2. Add a link to the playground https://huggingface.co/spaces/thinkall/AutoGen_Playground
4. Move examples to the ecosystem page 

## Related issue number

Closes #799
Closes #699
## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by skzhang1** [comment]
Screenshot 2023-11-28 at 22 34 05

**Comment by sonichi** [comment]
Please pre-commit for code formatting.

**Comment by skzhang1** [comment]
Screenshot 2023-11-29 at 12 08 40

**Comment by qingyun-wu** [comment]
- For each entry in "Ecosystem", use a link to the corresponding integration/documentation instead of directly copying the content.  
- Under "Examples" only include one single file (merge the content from the existing two files).

**Comment by qingyun-wu** [comment]
Did you update the PR, I don't see the "Resources" tab.

**Comment by skzhang1** [comment]
> Did you update the PR? I don't see the "Resources" tab.
Screenshot 2023-11-29 at 14 14 05

Added. Please take a look.

**Comment by qingyun-wu** [comment]
Add a resources page.

**Comment by qingyun-wu** [comment]
Have you made changes? If so, could you please share what it looks like?

File: website/docs/Ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Ecosystem
+
+This page lists libraries that have integrations with Autogen for LLM applications using multiple agents in alphabetical order. Including your own integration to this list is highly encouraged. Simply open a pull request with a few lines of text, see the dropdown below for more information.
+
+
+## [MemGPT & AutoGen](https://memgpt.readthedocs.io/en/latest/autogen/)
```
Comment (qingyun-wu) [inline website/docs/Ecosystem.md:6]:
```suggestion
## [MemGPT + AutoGen](https://memgpt.readthedocs.io/en/latest/autogen/)
```

Comment (skzhang1) [reply website/docs/Ecosystem.md]: Resolved

**Review by qingyun-wu — APPROVED** [review]

**Pull request merged**